project.clj

```diff
@@ -76,9 +76,12 @@
   :aot [#"game\.*"
         #"web\.*"
         #"tasks.fetch"
-        #"jinteki\.*"]
+        #"jinteki\.*"
+        #"web.core"]
```

Comment:
Isn't this handled by the `#"web\.*"` inclusion above?

Reply:
Sadly, it still generates a warning message if the `main` function isn't called out explicitly.

Reply:
But, yes, it was being included in the AOT from the regex.

Reply:
Oh hell yeah, that's annoyed me forever.